astroplan/constraints.py

```diff
@@ -997,24 +997,59 @@ def observability_table(constraints, observer, targets, times=None,
     return tab
 
 
-def _rescale_minmax(vals, min_val, max_val):
-    """ Rescale altitude into an observability score."""
-    rescaled = (vals - min_val) / (max_val - min_val)
-    below = rescaled < 0
-    above = rescaled > 1
-    rescaled[below] = 0
-    rescaled[above] = 1
+def _rescale_minmax(vals, worst_val, best_val, better_than=1, worse_than=0):
```

Comment:
I think this function should be public (remove the leading `_`, added to `__all__`) since it will be used in custom constraints.